JobConfig/cosmic/cosmic_s2_resampler_hi.fcl

```diff
@@ -0,0 +1,17 @@
+// Re-sample cosmic dsstops particles from stage-1
```

Comment:
Is 'hi' not the default?  If so, can this configuration be made the default, with 'lo' overriding it?

Reply:
cosmic_s2_resampler_all.fcl is default.